js/README.md

```diff
@@ -397,27 +397,88 @@ From ORT v1.13 onwards the 'full' ONNX Runtime package is used. It supports both
      - replace `com.microsoft.onnxruntime:onnxruntime-android` with `com.microsoft.onnxruntime:onnxruntime-mobile` in /js/react_native/e2e/android/app/build.gradle
      - replace `onnxruntime-c` with `onnxruntime-mobile-c` in /js/react_native/e2e/ios/Podfile
 
-   From `<ORT_ROOT>/js/react_native/e2e/android`, run e2e Android tests as follows,
+  - Run E2E Testing with Detox framework
 
-   ```sh
-   ./gradlew :app:connectedDebugAndroidTest
-   ```
+    When testing with integrated [Detox](https://wix.github.io/Detox/docs/next/introduction/getting-started) framework for Android and iOS e2e apps:
+    - Detox prerequisites:
 
-   From `<ORT_ROOT>/js/react_native/e2e/ios`, run e2e iOS tests as follows,
+      Install detox command line tools:
+      ```
+      yarn global add detox-cli
+      ```
+      Install applesimutils which is required by Detox to work with iOS simulators. (Requires a MacOS device)
+      ```
+      brew tap wix/brew
+      brew install applesimutils
+      ```
+    - Add [Jest](https://jestjs.io/) dependency and install Detox itself for e2e Detox project from project directory `<ORT_ROOT>/js/react_native/e2e`.
+      ```
+      yarn add "jest@^29" --dev
+      ```
+      At the end of this step, should be able to see jest written to `package.json` file under the project root folder.
 
-   ```sh
-   xcrun xcodebuild test -workspace OnnxruntimeModuleExample.xcworkspace -scheme OnnxruntimeModuleExample -destination 'platform=iOS Simulator,OS=latest,name=iPhone 13'
-   ```
+      Then install detox itself.
+      ```
+      yarn add detox --dev
+      ```
```

Comment:
now that jest and detox are included in yarn.lock, are these steps still necessary?